### PR TITLE
data: Migration script

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: indigo/management/commands/datamigrationcleancountry.py [hunk 1]
```diff
@@ -0,0 +1,212 @@
+import jsondataferret
+from django.core.management.base import BaseCommand
+from jsondataferret.pythonapi.newevent import NewEventData, newEvent
+
+from indigo.models import Project
+
+
+class Command(BaseCommand):
+    help = "Data Migration - Clean Country"
```
Comment (Lathrisk) [inline indigo/management/commands/datamigrationcleancountry.py:9]: I'm not sure if this really explains the command. Perhaps something about converting to country codes?

File: indigo/management/commands/datamigrationcleancountry.py [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+                                "value", ""
+                            )
+                        )
+                    )
+                    country_code = get_country_code_from_old_data(
+                        delivery_location.get("location_name", {}).get("value", ""),
+                        delivery_location.get("location_country", {}).get("value", ""),
+                    )
+                    if country_code:
+                        print("RESULT Will change country to " + country_code)
```
Comment (Lathrisk) [inline indigo/management/commands/datamigrationcleancountry.py:45]: Doesn't really matter, but perhaps this W shouldn't be capitalised.

**Review by Lathrisk — APPROVED** [review]
Optional comments, otherwise LGTM

File: indigo/management/commands/datamigrationcleancountry.py [hunk 3]
```diff
@@ -0,0 +1,212 @@
...
+        return "GB"
+    if "Australia" in location_name or "Australia" in location_country:
+        return "AU"
+    if "Bristol" in location_name or "Bristol" in location_country:
+        return "GB"
+    if "London" in location_name or "London" in location_country:
+        return "GB"
+    if "UK" in location_name or "UK" in location_country:
+        return "GB"
+    if "West Midlands" in location_name or "West Midlands" in location_country:
```
Comment (Lathrisk) [inline indigo/management/commands/datamigrationcleancountry.py:106]: Was the old data of a standard format or is case a possible issue here?

Comment (ghost) [reply indigo/management/commands/datamigrationcleancountry.py:106]: It was not of standard format - if you mean case as in upper/lower in "West Midlands" then it matches what explicit instance in old data.

File: indigo/management/commands/datamigrationcleancountry.py (hunk 1 above)
Comment (ghost) [reply indigo/management/commands/datamigrationcleancountry.py:9]: Point, but as this should be run once then deleted I'll just go with it

**Pull request merged**